configs/ci/roles/config-manager.json

```diff
@@ -0,0 +1,67 @@
+{
+  "roles": [
+    {
+      "name": "RHC Administrator",
+      "description": "Perform any operations on the service enablement dashboard",
+      "system": true,
+      "platform_default": false,
+      "version": 1,
+      "access": [
+        {
+          "permission": "config-manager:state:read"
+        },
+        {
+          "permission": "config-manager:state:write"
+        },
+        {
+          "permission": "config-manager:state-changes:read"
+        },
+        {
+          "permission": "inventory:*:read"
+        },
+        {
+          "permission": "playbook-dispatcher:run:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "key": "service",
+                "operation": "equal",
+                "value": "config-manager"
```

Comment:
@Jason-RH I had configured the service predicate value as `config_manager` in playbook-dispatcher. I can change that there or, if you don't mind, change it here so that we get a match.

Reply:
:+1: I can change it here